pkgs/os-specific/linux/btfs/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
 
   preInstall = ''
     substituteInPlace scripts/btplay \
-      --replace "/usr/bin/env python" "${python}/bin/python"
+      --replace "/usr/bin/env python" "${python3}/bin/python"
```

Comment:
${python.3.interpreter}

Reply:
@FRidh Is it `${python3.interpreter}`, right?

Reply:
ah, yes